src/agent/onefuzz-agent/src/tasks/analysis/generic.rs

```diff
@@ -46,21 +53,59 @@ pub async fn spawn(config: Config) -> Result<()> {
     config.analysis.init().await?;
     config.tools.init_pull().await?;
 
+    // setup reports
```

Comment:
This comment doesn't really give me any particular idea of what is supposed to happen below. Could you expand it a bit, maybe addressing our expectation for the corner case of "no syncdirs configured"?